.github/workflows/ci.yml

```diff
@@ -140,6 +140,26 @@ jobs:
           RUSTFLAGS: ${{ env.RUSTFLAGS }} -C target-feature=+cmpxchg16b
         if: matrix.target == '' || startsWith(matrix.target, 'x86_64')
 
+      # Sandy Bridge (the first Intel chip that introduced AVX)
+      - run: cargo test -vv --workspace --exclude bench --all-features $TARGET $BUILD_STD $DOCTEST_XCOMPILE
+        env:
+          RUSTDOCFLAGS: ${{ env.RUSTDOCFLAGS }} -C target-cpu=sandybridge
```

Comment:
(I initially used haswell here, but the test was failing because GitHub-hosted macOS runner uses older CPUs.)